### PR TITLE
Checking the qmat solution

### PR DESCRIPTION
**Pull request by mlgiordano1**
The Qmat matrix as computed in the for loop is also transposed with respect to how I computed the Qmat. 

I'm not sure how this lines up with the logits later in the code so I am going to check that now. But it doesn't appear as though the Qmat is ever transposed in the code, so its very possible that a quick flip of these might help!

more to follow

### CODE REVIEW

**Comment by mlgiordano1** [comment]
Ok so in this version I have made new Pmat's and Qmat's. I think this is correct. It leads to different logits in the Mplus .inp files. I wonder if you could check this in a small set of models? 

That said, let me hedge what I've done here...the main way I think I might be wrong here is if I am misunderstanding what is being fixed in the actual Mplus Mixture model. I am less sure of what l11 and l12 are supposed to represent in the Mplus model (That's certainly where my understanding is breaking down, happy to talk more about that). However, I feel pretty good that the new P and Q matrices match the formulas in the aspar&muthen paper.
